src/main.rs

```diff
@@ -0,0 +1,19 @@
+mod api_client;
+mod error;
+mod serde;
+mod types;
+use api_client::Client;
```

Comment:
this binary is external to the library so: 
```suggestion
use beacon_api_client::Client;
```